### DISCUSSION TITLE
How to change GlassCard Margin?

### DISCUSSION THREAD

**Discussion by GaborTheHutt** [kikipoulet/SukiUI#198] [Q&A] [answered]
Hi,

Thank you for the great work!

Is there a way to remove the margin of the GlassCards?
I tried to add rounded corners to my image this way.

![image](https://github.com/kikipoulet/SukiUI/assets/60019588/fa6358bb-c148-4c80-8886-dea2e521fbed)

`<StackPanel Margin="20,20,20,20">
<suki:GlassCard  MaxHeight="500" MaxWidth="500" ClipToBounds="True" >
<Image Source="{Binding CameraImage}"  MaxHeight="520" MaxWidth="520" ClipToBounds="True"/>
</suki:GlassCard>
<suki:GlassCard Classes="HeaderCard" Margin="20,20,20,20" Width="100" Height="100"  >
<ToggleSwitch  CommandParameter="{Binding IsChecked,RelativeSource={RelativeSource Self}}" Command="{Binding SwitchChangedCommand}" />
</suki:GlassCard>
</StackPanel>`

**Comment by AuroraZiling** [comment, marked as answer]
You could add property `Padding="0"` to `GlassCard`

```csharp
<controls:GlassCard
          ClipToBounds="True"
          MaxHeight="500"
          MaxWidth="500"
          Padding="0">
          ...
</controls:GlassCard>
```

**Reply by kikipoulet** [reply]
Moreover, if it's just to add rounded corner on your image, you can use a simple Border with the same properties, it will be a little lighter.
